release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -620,11 +620,20 @@ In the table, features are marked with the following statuses:
 |REM
 |
 
+|vSphere on HW version 13
+|GA
+|GA
+|DEP
+
 |====
 
 [id="ocp-4-9-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-9-vsphere-and-virtual-hardware-version-deprecations"]
+==== vSphere 6.7U2 and earlier cluster installation and virtual hardware version 13 are now deprecated
```

Comment:
Do we need to update the "hardware version 13" to "hardware version earlier than 15"? cc @gnufied

Reply:
Others lgtm

Reply:
@duanwei33 what hardware versions were supported for OCP 4.8? I was told that we are deprecating HW 13; stating that we're deprecating all hardware versions below 15 would imply that we supported multiple versions less than 15.